packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -52,144 +58,104 @@ const genQaMessageTypePayload = {
 };
 
 describe('quantic-generated-answer', () => {
+  beforeEach(() => {
+    cy.clock(0, ['Date']);
+  });
+
+  afterEach(() => {
+    cy.clock().then((clock) => {
+      clock.restore();
+    });
+  });
+
   const pageUrl = 's/quantic-generated-answer';
 
   function visitGeneratedAnswer(options: Partial<GeneratedAnswerOptions> = {}) {
     interceptSearch();
     cy.visit(pageUrl);
     configure(options);
+    if (options.useCase === useCaseEnum.insight) {
+      InsightInterfaceExpect.isInitialized();
+      performSearch();
+    }
```

Comment:
I think I would prefer this to be either in it's own method, or in the "beforeEach()" instead.
It's not really "visitGeneratedAnswer" it's a distinct behavior